SPECS/cert-manager/CVE-2024-51744.patch

```diff
@@ -0,0 +1,88 @@
+From 24c143f0c31c653523182e958d1e73c8e1022541 Mon Sep 17 00:00:00 2001
+From: jykanase <v-jykanase@microsoft.com>
+Date: Thu, 27 Mar 2025 11:36:04 +0000
+Subject: [PATCH] CVE-2024-51744
+
+Source Link: https://github.com/golang-jwt/jwt/commit/7b1c1c00a171c6c79bbdb40e4ce7d197060c1c2c#diff-83eb8e32639d01cf443d6d8bde24c1c8be78766090d8c5f8586c36250cfedca6R50-R51
```

Comment:
Patch applies with a minor shift
Patch is applied while building
There is only one instance of `parser.go` file present in the vendor directory.